advanced_search.info.yml

```diff
@@ -6,6 +6,6 @@ type: module
 package: Islandora
 core_version_requirement: ^8.8 || ^9 || ^10 || ^11
 dependencies:
-  - drupal:facets
-  - drupal:facets_summary
-  - drupal:search_api_solr
+  - facets:facets
+  - facets_summary:facets_summary
```

Comment:
```suggestion
  - facets:facets_summary
```

Reply:
Oh no, I didn't see @joecorall's comment when I hit approve! Does this need to get reverted or is there another way to handle this?

Reply:
I made a followup PR https://github.com/Islandora/advanced_search/pull/64

Reply:
https://github.com/Islandora/advanced_search/pull/65

Reply:
Thank you!